ajax/libs/event-drops/package.json

```diff
@@ -0,0 +1,26 @@
+{
+	"name": "event-drops",
+	"npmName": "event-drops",
+	"npmFileMap": [
+		{
+		  "basePath": "src",
```

Comment:
Please set `dist` as basePath instead. Files in `dist` are what we need.

Reply:
Hi, 
In the Github repository there is no "dist" directory hence I included "src" directory as basePath
If it is still wrong then I would gladly change it to "dist"